### DISCUSSION TITLE
Unraid Docker install failed

### DISCUSSION THREAD

**Discussion by PyreWilderness** [nextcloud/all-in-one#4941] [Questions] [answered]
### Steps to reproduce
1. Create Docker container per instructions
2. Apply Settings the command failed

### Expected behavior 
Docker container should start

### Actual behavior 
Docker command failed with this error docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data" to rootfs at "/mnt/docker-aio-config": stat /var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data: no such file or directory: unknown.

### Host OS 
Unraid 6.12.10

#### Nextcloud AIO version 
Can't access

#### Current channel 
latest

**Comment by szaimen** [comment, marked as answer]
> Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting

Hi, I found this: https://www.google.com/search?q=%22Error%20response%20from%20daemon%3A%20failed%20to%20create%20task%20for%20container%3A%20failed%20to%20create%20shim%20task%3A%20OCI%20runtime%20create%20failed%3A%20runc%20create%20failed%3A%20unable%20to%20start%20container%20process%3A%20error%20during%20container%20init%3A%20error%20mounting%22
